src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -118,7 +120,24 @@ public SdkResult GetSdkResult(int submissionId, SdkReference sdk, LoggingContext
                     SdkResultFactory resultFactory = new SdkResultFactory(sdk);
                     try
                     {
-                        SdkResult result = (SdkResult) sdkResolver.Resolve(sdk, context, resultFactory);
+                        SdkResult result;
+
+                        try
+                        {
+                            result = (SdkResult) sdkResolver.Resolve(sdk, context, resultFactory);
+                        }
+                        catch (Exception e) when (sdkResolver is NuGetSdkResolver && e is FileNotFoundException)
```

Comment:
Oof, this is specific. But I can't think offhand of a better way.

Reply:
Yeah we went back and forth on this.  We're explicitly adding the resolver so we felt comfortable with the special case here.  I'll add a comment to explain the reasoning